### PR TITLE
Fix Croatian localedata issues in mastodon.js

### PR DESCRIPTION
**Pull request by yookoala**
Resolves #2667.

(#2667 was updating against an older version of master branch. Not applicable anymore.)

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

File: app/javascript/mastodon/containers/mastodon.js [hunk 1]
```diff
@@ -44,25 +44,27 @@ import Mutes from '../features/mutes';
...
 import he from 'react-intl/locale-data/he';
+import hr from 'react-intl/locale-data/hr';
 import hu from 'react-intl/locale-data/hu';
+import id from 'react-intl/locale-data/id';
 import it from 'react-intl/locale-data/it';
 import ja from 'react-intl/locale-data/ja';
-import pt from 'react-intl/locale-data/pt';
 import nl from 'react-intl/locale-data/nl';
 import no from 'react-intl/locale-data/no';
+import oc from 'react-intl/locale-data/oc';
```
Comment (ykzts) [inline app/javascript/mastodon/containers/mastodon.js]: Is `react-intl/locale-data/oc` present?

**Review by ykzts — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/containers/mastodon.js (hunk 1 above)
Comment (ykzts) [inline app/javascript/mastodon/containers/mastodon.js]: > Error: Cannot find module 'react-intl/locale-data/oc'

via https://runkit.com/590de15c32efc70012d6d691/590de15c32efc70012d6d692

Comment (yookoala) [inline app/javascript/mastodon/containers/mastodon.js]: Removed Occitan reference. See aeaa4ad.

Comment (ykzts) [inline app/javascript/mastodon/containers/mastodon.js]: good.

**Review by ykzts — APPROVED** [review]

**Pull request merged**